controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -518,89 +487,6 @@ func (r *OCSInitializationReconciler) ensureOcsOperatorConfigExists(initialData
 	return nil
 }
 
-func (r *OCSInitializationReconciler) getEnableTopologyKeyValue() string {
-
-	for _, sc := range r.clusters.GetStorageClusters() {
-		if !sc.Spec.ExternalStorage.Enable && sc.Spec.ManagedResources.CephNonResilientPools.Enable {
-			// In internal mode return true even if one of the storageCluster has enabled it via the CR
-			return "true"
-		} else if sc.Spec.ExternalStorage.Enable {
-			// In external mode, check if the non-resilient storageClass exists
-			scName := util.GenerateNameForNonResilientCephBlockPoolStorageClass(&sc)
-			storageClass := util.GetStorageClassWithName(r.ctx, r.Client, scName)
-			if storageClass != nil {
-				return "true"
-			}
-		}
-	}
-
-	return "false"
-}
-
-// In case of multiple storageClusters when replica-1 is enabled for both an internal and an external cluster, different failure domain keys can lead to complications.
-// To prevent this, when gathering information for the external cluster, ensure that the failure domain is specified to match that of the internal cluster (sc.Status.FailureDomain).
-func (r *OCSInitializationReconciler) getTopologyDomainLabelsKeyValue() string {
-
-	for _, sc := range r.clusters.GetStorageClusters() {
-		if !sc.Spec.ExternalStorage.Enable && sc.Spec.ManagedResources.CephNonResilientPools.Enable {
-			// In internal mode return the failure domain key directly from the storageCluster
-			return sc.Status.FailureDomainKey
-		} else if sc.Spec.ExternalStorage.Enable {
-			// In external mode, check if the non-resilient storageClass exists
-			// determine the failure domain key from the storageClass parameter
-			scName := util.GenerateNameForNonResilientCephBlockPoolStorageClass(&sc)
-			storageClass := util.GetStorageClassWithName(r.ctx, r.Client, scName)
-			if storageClass != nil {
-				return getFailureDomainKeyFromStorageClassParameter(storageClass)
-			}
-		}
-	}
-
-	return ""
-}
-
-func (r *OCSInitializationReconciler) getEnableNFSKeyValue() string {
```

Comment:
Don't we need this anymore? How else would rook know to spin up the nfs daemons?